assets/facets.js

```diff
@@ -214,12 +252,19 @@ customElements.define('price-range', PriceRange);
 class FacetRemove extends HTMLElement {
   constructor() {
     super();
-    this.querySelector('a').addEventListener('click', (event) => {
-      event.preventDefault();
-      const form = this.closest('facet-filters-form') || document.querySelector('facet-filters-form');
-      form.onActiveFilterClick(event);
+    const facetLink = this.querySelector('a');
+    facetLink.setAttribute('role', 'button');
+    facetLink.addEventListener('click', this.closeFilter.bind(this));
+    facetLink.addEventListener('keyup', (event) => {
+      if (event.code.toUpperCase() == 'SPACE') this.closeFilter(event);
```

Comment:
Need to add `event.preventDefault()` here since pressing <kbd>Space</kbd> causes the page to scroll.